tests/data-test.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+HAPROXY_SOCKET_PATH="/opt/cardano/cnode/sockets/haproxy.socket"
+VENV_DIR_PATH="/opt/cardano/cnode/tests/koios-artifacts/tests/koios-tests"
+API_SPEC_PATH="/opt/cardano/cnode/tests/koios-artifacts/specs/results/koiosapi-guild.yaml"
+
+source "${VENV_DIR_PATH}/bin/activate"
+
+# This is pseudo-code, need to from haproxy:
+# 1) ssl vs no ssl servers
+# 2) correct port combination for server
+# 3) correct server name (for toggling MAINT state)
+# Below line just gets resolved server IP addresses into array
+servers=$(echo "show servers state" | nc -U "$HAPROXY_SOCKET_PATH" | grep grest_core | cut -d' ' -f5)
+mapfile -t serversArray <<<"${servers}"
+
+for server in "${serversArray[@]}"; do
+  echo "Running data tests for: $server"
+  pytest /opt/cardano/cnode/tests/koios-artifacts/tests \
+    --local-url https://eden-guildnet.koios.rest:8453/api/v0/ \
```

Comment:
We'll want to first run against 4 (A against B, if not matched - A against C, B against C - to stop at first match for 2 instances) trusted instances to form source of truth. If none seem to be in sync, we'd want to log an alert to be later consumed by notification service

Reply:
this is done now 👍